File: src/main/java/no/nav/domain/pensjon/regler/vedtak/VilkarsprovUforetrygdResultat.java
package no.nav.domain.pensjon.regler.vedtak;

import no.nav.domain.pensjon.regler.beregning2011.BeregningsresultatUforetrygd;

/**
 * @author Magnus Bakken (Accenture), PK-10597
 */
public class VilkarsprovUforetrygdResultat extends AbstraktVilkarsprovResultat {

    /**
     * Beregning av uføretrygden ved vilkårsprøving av halv minsteytelse.
     */
    private BeregningsresultatUforetrygd beregningsresultatUforetrygd;

    /**
     * Beregnet halv minsteytelse;
     */
    private int halvMinsteytelse;

}

File: src/main/java/no/nav/pensjon/regler/domain/vedtak/VilkarsprovUforetrygdResultat.kt
package no.nav.pensjon.regler.domain.vedtak

import no.nav.pensjon.regler.domain.beregning2011.BeregningsresultatUforetrygd

class VilkarsprovUforetrygdResultat : AbstraktVilkarsprovResultat() {
    /**
     * Beregning av uføretrygden ved vilkårsprøving av halv minsteytelse.
     */
    var beregningsresultatUforetrygd: BeregningsresultatUforetrygd? = null

    /**
     * Beregnet halv minsteytelse;
     */
    var halvMinsteytelse = 0

}